patient_mobile/lib/screens/simulation/chat_page.dart

```diff
@@ -16,13 +19,45 @@ class ChatPage extends StatefulWidget {
 
 class _ChatPageState extends State<ChatPage> {
   var sessionId = '';
+  bool isLoading = true;
+
+  List<dynamic> messages = [
+    [
+      'Bonjour, je m’appel Edgar et je serai votre assistant tout au long de cette simulation. Pour commencer, pouvez-vous me dire où vous avez mal ?',
+      false,
+    ]
+  ];
 
   final ScrollController _scrollController = ScrollController();
 
   @override
   void initState() {
     super.initState();
-    getSessionId();
+    loadSession();
+  }
+
+  Future<void> getStatusNlp() async {
+    await getNlpUp(context).then(
+      (value) {
+        if (value == true) {
+          setState(
+            () {
+              isLoading = false;
+            },
+          );
+        }
+      },
+    );
+  }
+
+  Future<void> loadSession() async {
+    await getSessionId();
+    Timer.periodic(
+      Duration(seconds: 8),
```

Comment:
Not that good, because even if the NLP is up the user needs to wait 8 seconds, get the status of the NLP first and IF it's down check every 8 seconds if it's uo